lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -127,6 +128,44 @@ def __init__(self, namespace: str):
         self.client = Client()
         self.namespace = namespace
 
+    def pod_is_ready(
+        self,
+        pod_name: str,
+        network_annotations: list[NetworkAnnotation],
+        containers_requiring_net_admin_capability: list[str],
+    ) -> bool:
```

Comment:
```suggestion
    def pod_is_ready(
        self,
        pod_name: str,
        *,
        network_annotations: list[NetworkAnnotation],
        containers_requiring_net_admin_capability: list[str],
    ) -> bool:
```

I would force the last 2 arguments to be used as keyword-only arguments for clarity.